pyproject.toml

```diff
@@ -69,7 +69,9 @@ arroyo = [
     "websockets",
     "zarr",
     "scikit-learn==1.3.0",
+    "torch>=2.0.0",
```

Comment:
Something to think about for a future PR is how to effectively handle changes in arroyo operators that require different frameworks, e.g. tensorflow vs pytorch, umap vs scikit-learn, etc.